client_end_user/shared/src/commonMain/kotlin/data/gateway/remote/FakeRemoteGateway.kt

```diff
@@ -9,6 +9,7 @@ import domain.entity.Offer
 import domain.entity.PriceLevel
 import domain.entity.Restaurant
 import domain.entity.Time
+import domain.entity.User
 import domain.gateway.IFakeRemoteGateway
 
 class FakeRemoteGateway : IFakeRemoteGateway {
```

Comment:
Why IFakeRemoteGateway !
we can use the real interfaces but different classes implementation